pkg/kubectl/cmd/edit.go

```diff
@@ -93,7 +93,7 @@ func NewCmdEdit(f cmdutil.Factory, out, errOut io.Writer) *cobra.Command {
 		Example: fmt.Sprintf(editExample),
 		Run: func(cmd *cobra.Command, args []string) {
 			options.ChangeCause = f.Command(cmd, false)
-			if err := options.Complete(f, out, errOut, args); err != nil {
+			if err := options.Complete(f, out, errOut, args, cmd); err != nil {
```

Comment:
> kubectl edit <resource>, by default don't include uninitialized objects, unless user explicitly sets --include-uninitialized=true.

@dixudx I think we need that flag per https://github.com/kubernetes/kubernetes/issues/49035#issuecomment-321615737. Did I miss some something here?